Import of zipped project causes 'Can't find project' ws-agent exception

### Description
We have next error message in `ws-agent` logs when importing zipped project through the REST API by using org.eclipse.che.api.project.server.ProjectService#importZip() method:
```
2018-02-12 23:45:29,273[nio-4401-exec-5]  [ERROR] [o.e.c.p.j.s.ProjectListeners 109]    - Can't find project /java-inner-lambda
org.eclipse.che.api.core.NotFoundException: Can't find project
```
<details>
 <summary>Full ws-agent log</summary>
 <pre>
2018-02-12 23:45:25,963[ost-startStop-1]  [INFO ] [i.WorkspaceProjectSynchronizer 62]   - Workspace ID: workspacee0ln0c54czdelsue
2018-02-12 23:45:25,968[ost-startStop-1]  [INFO ] [i.WorkspaceProjectSynchronizer 63]   - API Endpoint: http://che-host:8080/api
2018-02-12 23:45:26,393[rcherInitThread]  [INFO ] [o.e.c.a.s.s.i.LuceneSearcher 158]    - Initial indexing complete after 2 msec 
2018-02-12 23:45:29,273[nio-4401-exec-5]  [ERROR] [o.e.c.p.j.s.ProjectListeners 109]    - Can't find project /java-inner-lambda
org.eclipse.che.api.core.NotFoundException: Can't find project
	at org.eclipse.che.plugin.java.server.ProjectListeners.lambda$isJavaProject$0(ProjectListeners.java:105)
	at org.eclipse.che.plugin.java.server.ProjectListeners$$Lambda$138/1677568591.get(Unknown Source)
	at java.util.Optional.orElseThrow(Optional.java:290)
	at org.eclipse.che.plugin.java.server.ProjectListeners.isJavaProject(ProjectListeners.java:105)
	at org.eclipse.che.plugin.java.server.ProjectListeners.access$100(ProjectListeners.java:42)
	at org.eclipse.che.plugin.java.server.ProjectListeners$ProjectCreated.onEvent(ProjectListeners.java:118)
	at org.eclipse.che.plugin.java.server.ProjectListeners$ProjectCreated.onEvent(ProjectListeners.java:114)
	at org.eclipse.che.api.core.notification.EventService.publish(EventService.java:107)
	at org.eclipse.che.api.project.server.impl.ProjectServiceApi.importZip(ProjectServiceApi.java:520)
	at org.eclipse.che.api.project.server.ProjectService.importZip(ProjectService.java:528)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.everrest.core.impl.method.DefaultMethodInvoker.invokeMethod(DefaultMethodInvoker.java:140)
	at org.everrest.core.impl.method.DefaultMethodInvoker.invokeMethod(DefaultMethodInvoker.java:60)
	at org.everrest.core.impl.RequestDispatcher.doInvokeResource(RequestDispatcher.java:306)
	at org.everrest.core.impl.RequestDispatcher.invokeSubResourceMethod(RequestDispatcher.java:297)
	at org.everrest.core.impl.RequestDispatcher.dispatch(RequestDispatcher.java:233)
	at org.everrest.core.impl.RequestDispatcher.dispatch(RequestDispatcher.java:128)
	at org.everrest.core.impl.RequestHandlerImpl.handleRequest(RequestHandlerImpl.java:62)
	at org.everrest.core.impl.EverrestProcessor.process(EverrestProcessor.java:120)
	at org.everrest.core.servlet.EverrestServlet.service(EverrestServlet.java:61)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:742)
	at com.google.inject.servlet.ServletDefinition.doServiceImpl(ServletDefinition.java:286)
	at com.google.inject.servlet.ServletDefinition.doService(ServletDefinition.java:276)
	at com.google.inject.servlet.ServletDefinition.service(ServletDefinition.java:181)
	at com.google.inject.servlet.ManagedServletPipeline.service(ManagedServletPipeline.java:91)
	at com.google.inject.servlet.FilterChainInvocation.doFilter(FilterChainInvocation.java:85)
	at org.eclipse.che.plugin.activity.LastAccessTimeFilter.doFilter(LastAccessTimeFilter.java:52)
	at com.google.inject.servlet.FilterChainInvocation.doFilter(FilterChainInvocation.java:82)
	at org.eclipse.che.multiuser.machine.authentication.agent.MachineLoginFilter.doFilter(MachineLoginFilter.java:97)
	at com.google.inject.servlet.FilterChainInvocation.doFilter(FilterChainInvocation.java:82)
	at org.apache.catalina.filters.CorsFilter.handleNonCORS(CorsFilter.java:423)
	at org.apache.catalina.filters.CorsFilter.doFilter(CorsFilter.java:169)
	at org.eclipse.che.api.core.cors.CheCorsFilter.doFilter(CheCorsFilter.java:59)
	at com.google.inject.servlet.FilterChainInvocation.doFilter(FilterChainInvocation.java:82)
	at com.google.inject.servlet.ManagedFilterPipeline.dispatch(ManagedFilterPipeline.java:120)
	at com.google.inject.servlet.GuiceFilter.doFilter(GuiceFilter.java:135)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:199)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:96)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:140)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:81)
	at ch.qos.logback.access.tomcat.LogbackValve.invoke(LogbackValve.java:256)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:87)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:342)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:803)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:66)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:868)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1459)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:745)
 </pre>
</details>

The problem is with the method [org.eclipse.che.api.project.server.impl.ProjectServiceApi#importZip()](https://github.com/eclipse/che/blob/6.0.0/wsagent/che-core-api-project/src/main/java/org/eclipse/che/api/project/server/impl/ProjectServiceApi.java#L515) which publishes **ProjectCreatedEvent** before the project has been actually registered. As a result, [org.eclipse.che.plugin.java.server.ProjectListeners#isJavaProject()](https://github.com/eclipse/che/blob/6.0.0/plugins/plugin-java/che-plugin-java-ext-lang-server/src/main/java/org/eclipse/che/plugin/java/server/ProjectListeners.java#L102) can't find importing project in project registry (thanks @dkuleshov for investigation).

### Reproduction Steps
Execute one of selenium tests which imports the project at start (for example, **InnerClassAndLambdaDebuggingTest**), and read ws-agent logs from directory `/home/user/che/ws-agent/logs`.

**OS and version:**    
Eclipse Che 6.1.0-SNAPSHOT

--- a/wsagent/che-core-api-project/src/main/java/org/eclipse/che/api/project/server/impl/ProjectServiceApi.java
+++ b/wsagent/che-core-api-project/src/main/java/org/eclipse/che/api/project/server/impl/ProjectServiceApi.java
@@ -517,8 +517,6 @@ public class ProjectServiceApi {
 
     fsManager.unzip(wsPath, zip, skipFirstLevel);
 
-    eventService.publish(new ProjectCreatedEvent(wsPath));
-
     return Response.created(
             serviceContext
                 .getBaseUriBuilder()

